app/controllers/orders_controller.rb

```diff
@@ -34,6 +35,15 @@ def create
   def update_status
     order = current_user.orders.find(params[:id])
     order.update(status: "cancelled")
+
+    item_order = ItemOrder.where(order_id: params[:id])
+    item_order.each do |item|
+      item_to_restock = Item.find(item.item_id)
+      new_quantity = (item.quantity + item_to_restock.inventory)
+      item_to_restock.update(inventory: new_quantity)
```

Comment:
SImilar question here. I feel like there's a more direct way to do the math of `inventory +/- quantity`